Chargebee/Classes/Restore/CBPurchaseManager+Extension.swift

```diff
@@ -136,9 +136,13 @@ extension CBPurchase{
         for product in products {
             operationQueue?.addOperation{
                 if let _ = product.product.subscriptionPeriod {
-                    self.validateReceipt(product, completion: nil)
+                    if let customer = self.restoreCustomer {
+                        self.validateReceipt(product,customer: customer, completion: nil)
+                    }
```

Comment:
Can we pass the `customer` without having to unwrap? The `validateReceipt` takes in an optional param.

Reply:
Done Updated